fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -2,7 +2,7 @@
 
 Usability Guide
 
-So, what capabilities does 8Vim have? Once you know how to type with this thing (Learn how to type with this original [8Pen-game App] (https://play.google.com/store/apps/details?id=com.eightpen.android.wordcup&hl=en), you must know the following
+So, what capabilities does 8Vim have? Once you know how to type with this thing (Learn how to type with this original [8Pen-game video](https://www.youtube.com/watch?v=99vsUF4NuLk), you must know the following
```

Comment:
1. rechecking the whole contents:
    * i think you have missed that I have not changed just the link, but wording as well. 
    * It doesnt just says `[game](video)`, it says `[game video](video)`
    * ... and it seems pretty consistent to me, there's no mention of game/app elsewhere in this whole file

Some very slight improvements can be:

2. Being verbose, so:
    * `PR:` "8Pen-game video"
    *  `1 :` "8Pen-game's video"
    * ` 2 :` "8Pen-game intro video"

3. "with" --> "from"

```diff
- Learn ... with this ... video ...
+ Learn ... from this ... video ...
```

4. Can u please recheck regarding 1st point?? And also share your opinion on second point?